homeassistant/components/homekit/util.py

```diff
@@ -23,6 +24,21 @@ def validate_entity_config(values):
             raise vol.Invalid('The configuration for "{}" must be '
                               ' an dictionary.'.format(entity))
 
+        name = config.get(CONF_NAME)
+        params[CONF_NAME] = \
+            cv.string(name) if name else None
+
+        manufacturer = config.get(CONF_MANUFACTURER)
+        params[CONF_MANUFACTURER] = \
+            cv.string(manufacturer) if manufacturer else None
+
+        model = config.get(CONF_MODEL)
+        params[CONF_MODEL] = cv.string(model) if model else None
+
+        serial_number = config.get(CONF_SERIAL_NUMBER)
```

Comment:
The empty lines between these four config opions (`name` - `serial_number`) can be removed